### PR TITLE
Feature/config fix

### PR DESCRIPTION
**Pull request by stuaxo**
Fix references to 'ui' that should be 'config'

Readability fix for DEBUG setting check -

https://github.com/uktrade/invest/compare/feature/config-fix?expand=1#diff-1323dcc6d85cb5bceac7402cff7ddfa6L31

### CODE REVIEW

File: config/wsgi.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-WSGI config for ui project.
+WSGI config for config project.
```
Comment (konradko) [inline config/wsgi.py]: probably `invest` here

File: config/settings.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """
-Django settings for ui project.
+Django settings for config project.
```
Comment (konradko) [inline config/settings.py]: ditto

File: config/wsgi.py (hunk 1 above)
Comment (stuaxo) [reply config/wsgi.py]: Maybe the whole line is redundant -

We should already know we are in the "invest" project, by the github, and the README.

If you are looking for wsgi.py then you already know it is a wsgi config.

Comment (stuaxo) [reply config/wsgi.py]: I'm tempted to change to something like ```WSGI config``` or ```WSGI entrypoint``` so the text doesn't need to be updated for every project.

**Comment by stuaxo** [comment]
I just changed these to the suggested "blah blah invest project", realised I was bikeshedding :)

**Comment by stuaxo** [comment]
This has a couple of hacks, which will get removed in subsequent PRs once I start putting code here.

- a dummy unit test:  ```config/test_dummy.py``` pytest needs at least on unittest, and we want things to pass to make sure circle ci is setup.
- enable the admin site:  django needs at least one URL and this is a quick way of doing this (we probably will enable this anyway, but I haven't thought about it).

This adds migration as a step in the makefile.

**Comment by stuaxo** [comment]
@konradko is this good to go ?

**Comment by stuaxo** [comment]
Probably worth squashing-and-merging, not just merging.

File: config/settings.py [hunk 3]
```diff
@@ -28,7 +25,7 @@
 SECRET_KEY = os.environ["SECRET_KEY"]
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True if (os.getenv('DEBUG') == 'true') else False
+DEBUG = 'true' == os.getenv('DEBUG', '')
```
Comment (richtier) [inline config/settings.py]: the consensus seems to be placing the dynamic value before the static value - so to be consistent with other areas perhaps this should `DEBUG = os.getenv('DEBUG') == 'true'`

File: config/settings.py [hunk 4]
```diff
@@ -38,13 +35,24 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'whitenoise.runserver_nostatic',
```
Comment (richtier) [inline config/settings.py]: did not know about this. cool :+1:

File: config/settings.py [hunk 5]
```diff
@@ -38,13 +35,24 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
```
Comment (richtier) [inline config/settings.py]: `django.contrib.messages` is normally not needed, I suggest starting without it and add when you need it.

File: docker/env.test.json [hunk 6]
```diff
@@ -3,6 +3,7 @@
   "file_path": "./docker/.env.test",
   "env_vars": {
     "required": [
+      "SECRET_KEY"
```
Comment (richtier) [inline docker/env.test.json]: is `PORT` not required in dev mode?

File: makefile [hunk 7]
```diff
@@ -20,6 +20,7 @@ test:
 	$(COLLECT_STATIC) && $(FLAKE8) && $(PYTEST) && $(CODECOV)
 
 DJANGO_WEBSERVER := \
+    python manage.py migrate --noinput && \
```
Comment (richtier) [inline makefile]: loos like there is an indentation woe here.

File: makefile [hunk 8]
```diff
@@ -73,7 +74,6 @@ docker_build:
 
 DEBUG_SET_ENV_VARS := \
 	export PORT=8010; \
-	export SECRET_KEY=debug; \
```
Comment (richtier) [inline makefile]: the settings file treats `SECRET_KEY` as a required env var, so it looks like deleting this line should kill the app?

File: config/settings.py (hunk 3 above)
Comment (stuaxo) [reply config/settings.py]: My Java-dev history showing through here :D

File: config/settings.py (hunk 5 above)
Comment (stuaxo) [reply config/settings.py]: These were the ones from the default wagtail site, so I didn't look too closely, I'll try without messages.

**Comment by stuaxo** [comment]
Cheers, I'll fix these and merge.

File: makefile (hunk 7 above)
Comment (stuaxo) [reply makefile]: Good catch :D

**Pull request closed without merging**